modules/kubernetes-addons/istio/main.tf

```diff
@@ -0,0 +1,138 @@
+resource "kubernetes_namespace" "istio_system" {
+  metadata {
+    name = "istio-system"
+    labels = {
+      istio-injection = "enabled"
+    }
+  }
+}
+
+module "istio_base" {
+  source = "../helm-addon"
+  count  = var.install_istio_base ? 1 : 0
+
+  helm_config = merge(
+    {
+      name            = "istio-base"
+      chart           = "base"
+      repository      = "https://istio-release.storage.googleapis.com/charts"
+      version         = var.istio_version
+      namespace       = kubernetes_namespace.istio_system.metadata.0.name
+      timeout         = 120
```

Comment:
we can remove all of these from each one
- `timeout`
- `cleanup_on_fail`
- `force_update`
- `values`

unless there is a strong reason to deviate from the defaults